src/components/ApplicationDetails/ApplicationOverviewTab.tsx

```
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { useParams } from 'react-router-dom';
 import { RouterParams } from '../../routes/utils';
+import { ConformaViolationSummaryCard } from '~/components/Conforma/ConformaViolationSummaryCard';
```

ESLint

[error] `~/components/Conforma/ConformaViolationSummaryCard` import should occur before import of `../../routes/utils`

(import/order)